apps/angular/permissions/src/app/common/guards/is-admin.guard.ts

```diff
@@ -0,0 +1,18 @@
+import { inject } from '@angular/core';
+import { CanMatchFn } from '@angular/router';
+import { UserStore } from '../../user.store';
+import { map } from 'rxjs';
+import { TypedRoute } from '../../routes';
+
+export const HasPermission: CanMatchFn = (route: TypedRoute, segments) => {
+  const user$ = inject(UserStore).user$;
+  return user$.pipe(
+    map((user) => {
+      if (route?.data?.isAdmin) return user?.isAdmin ? true : false;
```

Comment:
don't need the true and false.

Reply:
yes, it doesn't need to be , i will fix it